error building ```opencv_contrib/modules/tracking/include/opencv2/tracking/tracker.hpp```

```
[ 98%] Built target opencv_test_matlab
[ 98%] Building CXX object modules/python2/CMakeFiles/opencv_python2.dir/__/src2/cv2.cpp.o
In file included from /home/epinux/dev/opencv_contrib/modules/tracking/include/opencv2/tracking/tracker.hpp:48:0,
                 from /home/epinux/dev/opencv/build2/modules/python2/pyopencv_generated_include.h:50,
                 from /home/epinux/dev/opencv/modules/python/src2/cv2.cpp:12:
/home/epinux/dev/opencv_contrib/modules/tracking/include/opencv2/tracking/onlineMIL.hpp:57:23: error: expected unqualified-id before ‘>’ token
 #define  sign(s)  ((s > 0 ) ? 1 : ((s<0) ? -1 : 0))
                       ^
/home/epinux/dev/opencv_contrib/modules/tracking/include/opencv2/tracking/onlineMIL.hpp:57:23: error: expected ‘)’ before ‘>’ token
/home/epinux/dev/opencv_contrib/modules/tracking/include/opencv2/tracking/onlineMIL.hpp:57:23: error: expected ‘)’ before ‘>’ token
modules/python2/CMakeFiles/opencv_python2.dir/build.make:296: recipe for target 'modules/python2/CMakeFiles/opencv_python2.dir/__/src2/cv2.cpp.o' failed
make[2]: *** [modules/python2/CMakeFiles/opencv_python2.dir/__/src2/cv2.cpp.o] Error 1
CMakeFiles/Makefile2:13133: recipe for target 'modules/python2/CMakeFiles/opencv_python2.dir/all' failed
make[1]: *** [modules/python2/CMakeFiles/opencv_python2.dir/all] Error 2
Makefile:160: recipe for target 'all' failed
make: *** [all] Error 2
epinux@Debian-70-wheezy-64-minimal:~/dev/opencv/build2$ 

```

i have same error so kindly any one help it ... then build INSTALL  project
**`Error	4	error LNK1104: cannot open file '..\..\lib\Debug\opencv_bioinspired310d.lib'	C:\opencv3.1\build\modules\bioinspired\LINK	opencv_test_bioinspired
`**
**Error	5	error LNK1104: cannot open file '..\..\lib\Debug\opencv_bioinspired310d.lib'	C:\opencv3.1\build\modules\bioinspired\LINK	opencv_perf_bioinspired
**

--- a/modules/tracking/include/opencv2/tracking/onlineMIL.hpp
+++ b/modules/tracking/include/opencv2/tracking/onlineMIL.hpp
@@ -54,8 +54,6 @@ namespace cv
 //TODO based on the original implementation
 //http://vision.ucsd.edu/~bbabenko/project_miltrack.shtml
 
-#define  sign(s)  ((s > 0 ) ? 1 : ((s<0) ? -1 : 0))
-
 class ClfOnlineStump;
 
 class CV_EXPORTS ClfMilBoost

--- a/modules/tracking/src/onlineMIL.cpp
+++ b/modules/tracking/src/onlineMIL.cpp
@@ -42,6 +42,8 @@
 #include "precomp.hpp"
 #include "opencv2/tracking/onlineMIL.hpp"
 
+#define  sign(s)  ((s > 0 ) ? 1 : ((s<0) ? -1 : 0))
+
 template<class T> class SortableElementRev
 {
  public:

